doc/source/whatsnew/v0.24.0.txt

```diff
@@ -337,7 +337,7 @@ Plotting
 ^^^^^^^^
 
 - Bug in :func:'DataFrame.plot.scatter' and :func:'DataFrame.plot.hexbin' caused x-axis label and ticklabels to disappear when colorbar was on in IPython inline backend (:issue:`10611` and :issue:`10678`)
--
+- Bug in :func:'DataFrame.plot.scatter' and :func:'DataFrame.plot.hexbin' resulted in misplaced colorbars when subplot was used (related to :issue:`20455`)
```

Comment:
you don’t need another entry, just add on to the previous one (just the number is ok)